tests/enterprise-contract/contract.go

```diff
@@ -343,8 +343,7 @@ var _ = framework.EnterpriseContractSuiteDescribe("Enterprise Contract E2E tests
 					Expect(fwk.AsKubeAdmin.TektonController.CreateOrUpdateSigningSecret(goldenImagePublicKey, secretName, namespace)).To(Succeed())
 					generator.PublicKey = fmt.Sprintf("k8s://%s/%s", namespace, secretName)
 
-					policy := contract.PolicySpecWithSourceConfig(
```

Comment:
The purpose of this test is to verify a list of images can be used. We don't need to use a custom policy config for this. The test is modified to simply use the default EC policy config.